nix/vast/default.nix

```diff
@@ -83,7 +83,7 @@ stdenv.mkDerivation rec {
     "-DVAST_ENABLE_BACKTRACE=ON"
     "-DVAST_ENABLE_JEMALLOC=ON"
     "-DVAST_ENABLE_LSVAST=ON"
-    "-DVAST_ENABLE_MDX_REGENERATE=ON"
+    "-DVAST_ENABLE_VAST_REGENERATE=OFF"
```

Comment:
This variable name change is in conflict with #2309, which uses `VAST_ENABLE_MDX_REGENERATE` still. We need to be careful when merging either of the PRs.